client/src/js/reducers/index.js

```diff
@@ -0,0 +1,18 @@
+// client/src/js/reducers/index.js
```

Comment:
Not sure I understand what this reducers file does, what do you need it for?

Reply:
reducer is the key of Redux, which tells the redux store how it generates a new state based on the action object dispatched.